wallet-backup.json

```diff
@@ -0,0 +1 @@
+{"@context":["https://www.w3.org/2018/credentials/v1","https://w3id.org/wallet/v1"],"id":"did:key:z6LSjTbRETJjUCDiQopbeCgZKRisy7mdchwiMBPTQktcibGh#encrypted-wallet","type":["VerifiableCredential","EncryptedWallet"],"issuer":"did:key:z6LSjTbRETJjUCDiQopbeCgZKRisy7mdchwiMBPTQktcibGh","issuanceDate":"2022-01-24T17:07:28.164Z","credentialSubject":{"id":"did:key:z6LSjTbRETJjUCDiQopbeCgZKRisy7mdchwiMBPTQktcibGh","encryptedWalletContents":{"protected":"eyJlbmMiOiJYQzIwUCJ9","recipients":[{"header":{"kid":"did:key:z6LSjTbRETJjUCDiQopbeCgZKRisy7mdchwiMBPTQktcibGh#z6LSjTbRETJjUCDiQopbeCgZKRisy7mdchwiMBPTQktcibGh","alg":"ECDH-ES+A256KW","epk":{"kty":"OKP","crv":"X25519","x":"4O7vuUD6WVp--g5PE9Y-SFYudGDyc09iDiYgPAl9nSk"},"apu":"4O7vuUD6WVp--g5PE9Y-SFYudGDyc09iDiYgPAl9nSk","apv":"ZGlkOmtleTp6NkxTalRiUkVUSmpVQ0RpUW9wYmVDZ1pLUmlzeTdtZGNod2lNQlBUUWt0Y2liR2gjejZMU2pUYlJFVEpqVUNEaVFvcGJlQ2daS1Jpc3k3bWRjaHdpTUJQVFFrdGNpYkdo"},"encrypted_key":"hHimM7y_eJyiwUGWsoOLWeYJY_c_11OcKVKLlNZ8j7GbyNSrZ-6B5A"}],"iv":"u8inDjybFjZhb5pM1uW1kiLJCq_QFlvR","ciphertext":"npjMblcLPADqht2C8j4vA3UwqjmF4aNGx6PyVip_fiEw3ea4iLhPet8rf3A4i6HbfYxU_SbQ7qNc-VM933UWCECPiBKj2f6dq61y47VPajAi062FoYPM_cr9i9H_VkYmDIT1M2HWOoSpIpBjOpe1zJ7CwxeQkRB83vsNvCOrrNBZAKNT1FqF7QfII_QxBF9c-Xp6H3Q_Omggf4u0ictzfi0YH3QUBEXSOmttqag4SziFDxWk0QZqSdkI8x6wmzxLBGJR84lWMU5uWCE-nJ0yyvPT6TIelP_K0eBwUYCZHWHStG3mkSjxG1Wqr9KV8g9XSDTwphYHrnq8mo3DjQRuSsDViIlYdh3kqeRHOOIl71YFzU9jPq5gSgkDyF-SZKsKDdu7ZSWGeHZGAL-kdMAYO3fW2IudECuwhNgDefrcPUvsWoySxvVGuR2TCGJwl3UVHyH5ivMBhnh69w1IK9Z4k2qx8DyoAhti6CG5tdWIsZ8N7Cfl4R4nkV9IPj4pznBWjnZ5r6bIyMmHPtN42u_u_WBqT3qc9XXCZoVDeLIeDr5C4RKVb_yAAhXyl5P8Bpec0aOTKBpfftlXAfa0KgW7jqq8PW_D3dTxExxSp0_wLhz6xyMvVTKg6HvxP2YbzM2B41VkmSKhnhpIVpioU7JLYT33lf_Lf6ptIR4yexwydC80J2od_zVV341OoXl0gOLKnInRkyMbDVb7nQZujHkdjZucrX4oaW_sAcdDxQ0P-nJJw-Fffrw7gCxCXx1LEfAM3dhbsr5cCOtKm9Wt-08b_kmDtme9G-eRm4HrzjwULUU8oAByW8ctiXHJx2LTy7D7Ngr-x9C7XIdf5lqrzGDD_FKX0vPurKmVEw8sts8wZS59lPRFxMitcTtvUKQVtewhmW90oD3WY95ivzhGjn2sVX2vCY1xlqTF-Jtkgc0RP7hW-wGvWIG6rhYRUHk0l4r31uqQxbgsJ74BjYXgV2ue-7mtDXXAMLdhAmZj-V3EAXUxFby14YjyEL-By9_K7mB3usJWkRTbIRLpJhKcFo_0Dp6nfTrqkfXK_riusWsVuDHmORArG14DyUSQKP6k5qKMrYZyqyi4aM9rqEeX4qIkszZMp4Q0mrCR2Oz8UogqtpPuz2tN8VkYSw6HQQXBuw4UEmXGhz8rrd7s3AhfGgXR-04JscbO0M---oXOZqRRch2jcfBxLKnyWgbrZ0BfZ0vFXwzXDIyyOZCFRC3rK_hI1dJ-r3b_AJT6AHKl4lL1NkkAeoE1Ap_U7odMHu03N4T3R0vT3AMBpBcnJRGLOkSUjdovgJfWUp5LJtdnjFYIkHEqPMt5YwarQQY9uUiMw0jDUCF6ZoDjkj3GvujmuIjjbTO8gjr1zADC9I1RwFTniHZ0pcLO0ByZWCD4Q2jAslrA7tM9MgXi65KRwEamtYD5pjaUt4X2_BDG2Q7n4xVFm5araiQM2NCMAVuPBPY0JJCCc9KCi5L5TWbR3GdtxCKrpwDbOqTJZDcXA2XbtjIAgcKgWhT3tQPgRIG1f9YrT22rNIsY","tag":"icapQCdbb86Vmi9DC6vqww"}}}
```

Comment:
is this here intentionally?

Reply:
Yes, it will be used to test the wallet import flow

Reply:
there are two of these files in this change-set. Is one a duplicate?

Reply:
i've removed the top level one. Will move it to the test folder as a fixture

Reply:
fixed